lib/cart.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "store"
+require_relative "billing"
+
+
+class Cart
+  attr_reader :item_names, :order, :calculator
+
+  def initialize(calculator)
+    @order = Hash.new(0)
+    @item_names = Store::ITEMNAMES
+    @calculator = calculator
```

Comment:
instead of calculator being a part of Cart, we could decouple them.
in application.rb:

```
cart_1 = Cart.new()
order_1 = cart_1.enter_order  # could return something like an order

bill_1 = BillingMachine.new(order_1) #could init a new bill
bill_1.checkout #could handle all logic & calculation.
bill_1.reciept #could print the table with pricing
```